### ISSUE TITLE
Missing 400 response body structure

### ISSUE THREAD

**Issue by markedwards** [blomqma/next-rest-framework#170]
The generated OpenAPI spec describes 400 error responses as having this structure:
```
{
  "type": "object",
  "properties": { "message": { "type": "string" } },
  "required": ["message"],
  "additionalProperties": false
}
```

They actually include error output from zod:
```
{
  "message": "Invalid request body.",
  "errors": [
    {
      "code": "invalid_string",
      "validation": "datetime",
      "message": "Invalid datetime",
      "path": [
        "foo"
      ]
    }
  ]
}
```

Ideally the OpenAPI spec should include this so generated clients can access the errors.

**Comment by blomqma** [comment]
Thanks, this should be fixed in `v6.0.2`.

**Comment by markedwards** [comment]
Sorry, I think this is a duplicate of #175
